TF/src/main.py

```diff
@@ -30,21 +33,23 @@ def run_inference(filename, model_name, category, is_saved_model):
     # is_saved model is either True or False, denoting whether the .pb file 
     # is a frozen graph or saved model
     parser = argparse.ArgumentParser()
-    parser.add_argument('filename')
-    parser.add_argument('model_name')
-    parser.add_argument('category') 
-    parser.add_argument('is_saved_model')
+    parser.add_argument('--filename')
+    parser.add_argument('--model_name')
+    parser.add_argument('--category') 
+    parser.add_argument('--is_saved_model')
+    parser.add_argument('--input_operation_names', nargs="+", default = [])
```

Comment:
Are all these arguments "required" argument? Can we add some default values for them?

Reply:
Added default "True" to is_saved_model and "None" to category, filename and model_name are required.